pkg/status/controller.go

```diff
@@ -46,16 +45,14 @@ import (
 
 const controllerName = "Status"
 
-// Status controller watches workstatues and checks whether the corresponding
+// Controller watches workstatues and checks whether the corresponding
 // workload object asks for the singleton status returning. If yes,
 // the full status will be copied to the workload object in WDS.
 type Controller struct {
 	logger             logr.Logger
 	wdsName            string
 	wdsDynClient       dynamic.Interface
-	wdsKubeClient      kubernetes.Interface
 	itsDynClient       dynamic.Interface
-	itsKubeClient      kubernetes.Interface
```

Comment:
FYI, all the changes in this file are cleanup that can be done in an independent PR.